external-docs/docs/guides/errors/inspect-command-errors/unexpected-sparql-ask-query-response-type.md

```diff
@@ -0,0 +1,9 @@
+# Error - Unexpected sparql ASK query response type `<response_type>`.
+
+## When it occurs
+
+The inspect command encountered an unexpected response type `<response_type>` when running a sparql query.
```

Comment:
Can you provide information in the exception about which sparql query has the problem?